pubspec.yaml

```diff
@@ -9,11 +9,9 @@ environment:
   sdk: '>=2.18.0 <3.0.0'
 
```

Comment:
This change looks great. Should we bump the version now?

Reply:
Probably, but need to research more to figure out if this is landable first.